### PR TITLE
certificate-authority: set CN to owner when CSR CN is empty

### PR DESCRIPTION
**Pull request by jkralik**

### CODE REVIEW

File: pkg/security/certificateSigner/certificateSigner.go [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+type CertificateSigner struct {
+	caCert []*x509.Certificate
+	caKey  crypto.PrivateKey
+	cfg    SignerConfig
+}
+
+func New(caCert []*x509.Certificate, caKey crypto.PrivateKey, opts ...Opt) *CertificateSigner {
+	cfg := SignerConfig{
+		ValidNotAfter: time.Unix(1<<63-62135596801, 999999999),
```
Comment (Danielius1922) [inline pkg/security/certificateSigner/certificateSigner.go]: Just out of curiousity - what time is this?

Comment (jkralik) [reply pkg/security/certificateSigner/certificateSigner.go]: Is MaxTime - golang doesn't have constant 😭

Comment (jkralik) [reply pkg/security/certificateSigner/certificateSigner.go]: 292277024627-12-06 15:30:07.999999999 +0000 UTC

Comment (Danielius1922) [reply pkg/security/certificateSigner/certificateSigner.go]: I learn something everyday. :D

**Review by Danielius1922 — APPROVED** [review]

**Pull request merged**